src/utils/PageParsing/Nuget.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {readFileSync} from 'fs';
+import {join} from 'path';
+import {DATA_SOURCES, FORMATS, RepoType} from '../Constants';
+import {getArtifactDetailsFromDOM} from '../PageParsing';
+
+describe('Nuget Page Parsing', () => {
+  test('should parse a valid Nuget page', () => {
+    const html = readFileSync(join(__dirname, 'testdata/nuget.html'));
+
+    window.document.body.innerHTML = html.toString();
```

Comment:
For Nuget we actually don't need to test the html! There's nothing on the webpage we look at, as it were. However, I like having this file around (in case we need to later), so I'm going to leave it, just wanted to call this out.